packages/fields/src/types/Content/Implementation.js

```diff
@@ -99,7 +99,7 @@ async function processSerialised(document, blocks, graphQlArgs) {
 }
 
 export class Content extends Relationship {
-  constructor(path, fieldConfig, listConfig) {
+  constructor(path, { blocks, ...fieldConfig }, listConfig) {
```

Comment:
Can minimise the diff a bit here by renaming the input parameter:

```suggestion
  constructor(path, { blocks: inputBlocks, ...fieldConfig }, listConfig) {
```